src/Microsoft.Xaml.Behaviors/Install.ps1

```diff
@@ -0,0 +1,5 @@
+param($installPath, $toolsPath, $package, $project)
+
+# Remove the reference to the .Design.dll, which is incorrectly referenced during
+# the NuGet package installation in .NET Framework applications (not .NET Core).
+$project.Object.References | Where-Object { $_.Name -eq 'Microsoft.Xaml.Behaviors.Design' } | ForEach-Object { $_.Remove() }
```

Comment:
Make sure you test the uninstall scenario of this package in .NET Framework packages.config projects.

Reply:
Works as expected.